teps/0060-remote-resource-resolution.md

```diff
@@ -319,19 +324,252 @@ Why should this TEP _not_ be implemented?
 
 ## Alternatives
 
-<!--
-What other approaches did you consider and why did you rule them out?  These do
-not need to be as detailed as the proposal, but should include enough
-information to express the idea and why it was not acceptable.
--->
-
-## Infrastructure Needed (optional)
-
-<!--
-Use this section if you need things from the project/SIG.  Examples include a
-new subproject, repos requested, github details.  Listing these here allows a
-SIG to get the process for these resources started right away.
--->
+### Do Nothing
+
+Users wanting to refer to tasks in git (or anywhere else) are required to figure out their own
+implementation to fetch those resources, apply them, and use them in PipelineRuns / TaskRuns.
+
+Pros:
+- Pipelines has no work to do :tada:
+- Pipelines _could_ offer some supporting docs to help people do this.
+
+Cons:
+- No opportunity to standardize around remote fetching outside the sphere of tekton bundles.
+- If Pipelines did offer docs for people constructing their own solution then they'd need to be kept up to date.
+
+### Just Implement Git Support Directly in Pipelines
```

Comment:
This option is not necessarily orthogonal to the main proposal in this TEP, and it could in fact be a first step towards it.

Reply:
This is a great point.  I was considering this "all or nothing" but you're right.  We could absolutely treat this as a first step.

Reply:
Updated doc to call out this potential stepping-stone approach.